spec/javascripts/components/layout-super-navigation-header-spec.js

```diff
@@ -5,223 +5,664 @@ describe('The super header navigation', function () {
-                        '<a class="govuk-link gem-c-layout-super-navigation-header__dropdown-list-item-link" href="/search/policy-papers-and-consultations">' +
-                          'Policy papers and consultation' +
-                        '</a>' +
-                        '<p class="govuk-body-s gem-c-layout-super-navigation-header__dropdown-list-item-description">Consultations and strategy</p>' +
-                      '</li>' +
-                      '<li class="gem-c-layout-super-navigation-header__dropdown-list-item">' +
-                        '<a class="govuk-link gem-c-layout-super-navigation-header__dropdown-list-item-link" href="/search/transparency-and-freedom-of-information-releases">' +
-                          'Transparency' +
-                        '</a>' +
-                        '<p class="govuk-body-s gem-c-layout-super-navigation-header__dropdown-list-item-description">Government data, freedom of information releases and corporate reports</p>' +
-                      '</li>' +
-                  '</ul>' +
-                  '<ul class="govuk-list">' +
-                      '<li class="gem-c-layout-super-navigation-header__dropdown-list-item">' +
-                        '<a class="govuk-link gem-c-layout-super-navigation-header__dropdown-list-item-link" href="/government/how-government-works">' +
-                          'How government works' +
-                        '</a>' +
-                      '</li>' +
-                      '<li class="gem-c-layout-super-navigation-header__dropdown-list-item">' +
-                        '<a class="govuk-link gem-c-layout-super-navigation-header__dropdown-list-item-link" href="/government/get-involved">' +
-                          'Get involved' +
-                        '</a>' +
-                      '</li>' +
-                  '</ul>' +
-              '</div>' +
-              '</li>' +
-              '<li class="govuk-header__navigation-item gem-c-layout-super-navigation-header__item gem-c-layout-super-navigation-header__item--search">' +
-                '<a class="govuk-header__link gem-c-layout-super-navigation-header__item-link gem-c-layout-super-navigation-header__item-link--search" href="/search">' +
-                  '<span class="gem-c-layout-super-navigation-header__item-link-text--search">Search</span>' +
-                  '<svg class="gem-c-layout-super-navigation-header__item-link-icon--search" width="27" height="27" viewBox="0 0 27 27" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false">' +
-                    '<circle cx="10.0161" cy="10.0161" r="8.51613" stroke="currentColor" stroke-width="3" />' +
-                    '<line x1="15.8668" y1="16.3587" x2="25.4475" y2="25.9393" stroke="currentColor" stroke-width="3" />' +
-                  '</svg>' +
-                '</a>' +
-              '</li>' +
-            '</ul>' +
-          '</nav>' +
-        '</div>' +
-      '</header>'
+      `<nav
```

Comment:
Note - I need to remove the backticks and change to the `''+` method for backwards compatibility.

Reply:
You're a better reviewer than I!

Reply:
Done in https://github.com/alphagov/govuk_publishing_components/pull/2223/commits/7b930cdc1dd80463d0ce2943c713d94b2c5f427b.